python/tests/global/test_formatting.py

```diff
@@ -0,0 +1,35 @@
+import sys
+import os
+
+import pytest
+
+from .excluded_files import get_excluded_files
+
+
+def get_py_files(path):
+    fnames = []
+    for root, dirs, files in os.walk(path):
+        for file in files:
+            if file.endswith(".py"):
+                fnames.append(os.path.join(root, file))
+    return fnames
+
+
+@pytest.mark.skipif(sys.version_info < (3, 6), reason="requires Python3")
+def test_code_style():
+    from click.testing import CliRunner
+    import black
+    from pathlib import Path
+
+    root = str(Path(__file__).parent.parent.parent)
```

Comment:
Could you use `tests.conftest.source_root` here? This bars refactoring of this file.

Reply:
Sure, do you agree with something like this: `root = os.path.join(source_root(), "python")`?

Reply:
Yes, totally. This means we can move `test_formatting.py` out of `global` without having to edit the contents of the file.

Reply:
Agreed, moved it out of `global` now